src/Microsoft.TestPlatform.Extensions.HtmlLogger/HtmlLogger.cs

```diff
@@ -282,7 +282,7 @@ public void TestRunCompleteHandler(object sender, TestRunCompleteEventArgs e)
                 PassedTests = PassedTests,
                 TotalTests = TotalTests,
                 SkippedTests = SkippedTests,
-                PassPercentage = (PassedTests * 100) / TotalTests,
+                PassPercentage = (PassedTests * 100) / (TotalTests >= 1 ? TotalTests : 1),
```

Comment:
Would be better to add some comment explaining DivideByZero case, cause it is not obvious why we change TotalTests  = 1. Or change the implementation to make PassPercentage = 0 when TotalTests  = 0.

Reply:
Good point, I think I had it done the second way before, but lost the commit somewhere. Will change to return 0 when total tests is 0.